docs/README.md

```diff
@@ -39,7 +39,7 @@ The `add-to-cart-button` is a block responsible for adding products in the [Mini
 
 | Prop name               | Type      | Description                                                                       | Default value        |
 | ----------------------- | --------- | --------------------------------------------------------------------------------- | -------------------- |
-| `onClickBehavior`       | `enum` | Controls what happens when users click the button. Possible values are: `go-to-product-page` or `add-to-cart`. | `add-to-cart`              |
+| `onClickBehavior`       | `enum` | Controls what happens when users click the button. Possible values are: `go-to-product-page`, `add-to-cart` and `go-to-product-page-multiple-available-skus`. | `add-to-cart`              |
```

Comment:
Suggestion on the name for this option are more than welcome 😅

Reply:
ensure-sku-selection